public/pages/Reindex/components/IndexSelect/IndexSelect.test.tsx

```diff
@@ -12,19 +12,18 @@ import { act } from "react-dom/test-utils";
 
 describe("<IndexSelect /> spec", () => {
   it("renders the component", async () => {
-    act(() => {
-      const component = render(
-        <CoreServicesContext.Provider value={coreServicesMock}>
-          <IndexSelect
-            getIndexOptions={(str) => Promise.resolve([{ label: "sourceIndex" }])}
-            onSelectedOptions={(options) => {}}
-            selectedOption={[{ label: "sourceIndex" }]}
-            singleSelect={true}
-          />
-        </CoreServicesContext.Provider>
-      );
-      expect(component).toMatchSnapshot();
-    });
+    const component = render(
+      <CoreServicesContext.Provider value={coreServicesMock}>
+        <IndexSelect
+          getIndexOptions={(str) => Promise.resolve([{ label: "sourceIndex" }])}
+          onSelectedOptions={(options) => {}}
+          selectedOption={[{ label: "sourceIndex" }]}
+          singleSelect={true}
+        />
+      </CoreServicesContext.Provider>
+    );
+    await waitFor(() => {});
```

Comment:
Put assert inside waitFor block is better, but it's ok to leave it as it is as long as the UT can pass consistently.

Reply:
for now, i din't see any failed run for this UT. let's keep it as it is.